iModelCore/ECPresentation/Tests/NonPublished/Integration/Content/Content_RelatedPropertiesTests.cpp

```diff
@@ -1396,53 +1466,104 @@ TEST_F(RulesDrivenECPresentationManagerContentTests, LoadsXToManyRelatedInstance
 /*---------------------------------------------------------------------------------**//**
 * @bsitest
 +---------------+---------------+---------------+---------------+---------------+------*/
-DEFINE_SCHEMA(LoadsXToManyRelatedInstancesAsArrays_MergesArrayValuesWhenArraySizeIsOneElementAndValuesAreEqual, R"*(
-    <ECEntityClass typeName="A">
-        <ECProperty propertyName="StringProperty" typeName="string" />
+DEFINE_SCHEMA(LoadsXToManyRelatedInstancesAsArrays_WithCustomDisplayLabels, R"*(
+    <ECEntityClass typeName="ChildClass1">
+        <ECNavigationProperty propertyName="Parent" relationshipName="ParentHasChildren1" direction="Backward" />
     </ECEntityClass>
-    <ECEntityClass typeName="B">
+    <ECStructClass typeName="MyStruct">
         <ECProperty propertyName="IntProperty" typeName="int" />
-        <ECProperty propertyName="LongProperty" typeName="long" />
-        <ECNavigationProperty propertyName="A" relationshipName="A_B" direction="Backward" />
+        <ECProperty propertyName="StringProperty" typeName="string" />
+    </ECStructClass>
+    <ECEntityClass typeName="ChildClass2">
+        <ECNavigationProperty propertyName="Parent" relationshipName="ParentHasChildren2" direction="Backward" />
+        <ECProperty propertyName="IntProperty" typeName="int" />
+        <ECProperty propertyName="StringProperty" typeName="string" />
+        <ECArrayProperty propertyName="ArrayProperty" typeName="int" />
+        <ECStructProperty propertyName="StructProperty" typeName="MyStruct" />
+        <ECStructArrayProperty propertyName="StructArrayProperty" typeName="MyStruct" />

```

Comment:
do we need all these different kinds of properties if we're testing display labels in this test?

Reply:
Removed